### PR TITLE
Misc. doxy and source comment typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 --skip="*.ts" -I ../shotcut-word-whitelist.txt` where whitelist consist of:
```
lod
ot
seeked
```

### CODE REVIEW

File: CuteLogger/src/AbstractAppender.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@
  * imagine.
  *
  * For the simple non-structured plain text logging (for example, to a plain text file or to the console output) you may
- * like to subclass the AbstractStringAppender instead of AbstractAppender, which will give you a more convinient way to
+ * like to subclass the AbstractStringAppender instead of AbstractAppender, which will give you a more convenient way to
```
Comment (ddennedy) [inline CuteLogger/src/AbstractAppender.cpp]: CuteLogger is not our code; it is imported from another project. I do not want to revise all their code like this.

File: drmingw/doc/README.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ To install enter
 
     drmingw -i
 
-Dr. Mingw will register itself as the JIT debugger by writting into the system registry. Make sure you have Administrator rights. See [this page](http://msdn.microsoft.com/en-us/library/windows/desktop/bb204634.aspx) for more information on how this works.
```
Comment (ddennedy) [inline drmingw/doc/README.md]: drmingw is also imported code that I do not want to update like this.

File: src/qml/export-edl/export-edl.js [hunk 3]
```diff
@@ -4275,7 +4275,7 @@ process.nextTick = function (fun) {
     }
 };
 
-// v8 likes predictible objects
```
Comment (ddennedy) [inline src/qml/export-edl/export-edl.js]: export-edl.js is a generate file where this line came from a node.js npm module - more code I do not want to maintain.

**Comment by luzpaz** [comment]
Will revise the commit

**Comment by luzpaz** [comment]
Commit revised

**Pull request merged**